docs/prepare-environment/custom-env.txt

```diff
@@ -59,96 +59,97 @@ It is also possible to set these variables for each accelerator type separately:
    python3 -m pip install pandas
 
-The :download:`Iris example </examples/iris_tf_keras.tgz>` contains a TensorFlow Keras model that
-uses a startup hook to install an additional Python dependency.
+And, this :download:`Iris example </examples/iris_tf_keras.tgz>` contains a TensorFlow Keras model
+that uses a startup hook to install an additional Python dependency.
 
 .. _container-images:
 
 ******************
  Container Images
 ******************
 
-Determined provides a set of officially supported Docker images. These are the default images used
-to launch containers for experiments, commands, and any other workflow in the Determined system.
+Officially supported, default Docker images are provided to launch containers for experiments,
+commands, and other workflows.
+
+All trial runner containers are launched with additional Determined-specific harness code, which
+orchestrates model training and evaluation in the container. Trial runner containers are also loaded
+with the experiment model definition and hyperparameter values for the current trial.
+
+GPU-specific versions of each library are automatically selected when running on agents with GPUs.
 
 .. _default-environment:
 
 Default Images
 ==============
 
-In the current version of Determined, experiments and tasks are executed in containers with the
-following:
-
--  Ubuntu 18.04
--  CUDA 11.1
--  Python 3.8.x
--  TensorFlow 2.4.x
--  PyTorch 1.9.x
-
-Determined will automatically select GPU-specific versions of each library when running on agents
-with GPUs.
-
-In addition to the above settings, all trial runner containers are launched with additional
-Determined-specific harness code that orchestrates model training and evaluation in the container.
-Trial runner containers are also loaded with the experiment's model definition and values of the
-hyperparameters for the current trial.
-
-.. note::
-
-   The default images are
-   ``determinedai/environments:cuda-11.3-pytorch-1.10-lightning-1.5-tf-2.8-gpu-0.17.15`` and
-   ``determinedai/environments:py-3.8-pytorch-1.10-lightning-1.5-tf-2.8-cpu-0.17.15`` for GPU and
-   CPU respectively.
++-------------+---------------------------------------------------------------------------------------+
+| Environment | File Name                                                                             |
++=============+=======================================================================================+
+| CPUs        | ``determinedai/environments:py-3.8-pytorch-1.10-lightning-1.5-tf-2.8-cpu-0.17.12``    |
++-------------+---------------------------------------------------------------------------------------+
+| Nvidia GPUs | ``determinedai/environments:cuda-11.3-pytorch-1.10-lightning-1.5-tf-2.8-gpu-0.17.12`` |
++-------------+---------------------------------------------------------------------------------------+
+| ROCm        | ``determinedai/environments:rocm-4.2-pytorch-1.9-tf-2.5-rocm-0.17.12``                |
```

Comment:
using either CUDA & ROCm or Nvidia GPUs & AMD GPUs dichotomy would probably be more correct

Reply:
@shiyuann indicated these versions were the same as listed in the Experiment Configuration _image_ field (likely an oversight). I'll make it 0.17.15 in both locations. Eventually, there should be only one reference, not duplicated.

Reply:
@ioga  Not sure what you mean here. These 3 were taken from the Experiment Configuration _image_ documentation.

Reply:
@ioga Please, disregard. I see what you mean.